lib/subscriber_list_mover.rb

```diff
@@ -48,4 +53,30 @@ def call
 
     puts "#{subscribers.count} active subscribers moved from #{from_slug} to #{to_slug}"
   end
+
+  def email_change_to_subscribers(source_subscriber_list)
+    email_subject = "Changes to GOV.UK email alerts"
+    email_utm_parameters = {
+        utm_source: email_subject,
```

Comment:
I think we should try to use a more unique identifier here, as for [content changes](https://github.com/alphagov/email-alert-api/blob/cb0575b58b7cfc764c99fdaac0ade6c9e305eb74/app/presenters/content_change_presenter.rb#L33). This means we could filter for this specific change in GA.

How about the `from_slug`?

Reply:
[Done](https://github.com/alphagov/email-alert-api/commit/597fc4aeaf9d70896b7c826d0b2716d3c560f045)